ffi.go

```diff
@@ -5,12 +5,15 @@ package wasmtime
 // #cgo windows CFLAGS:-DWASM_API_EXTERN= -DWASI_API_EXTERN=
 // #cgo windows LDFLAGS:-lwasmtime -luserenv -lole32 -lntdll -lws2_32 -lkernel32 -lbcrypt
 // #cgo linux,amd64 LDFLAGS:-L${SRCDIR}/build/linux-x86_64
+// #cgo linux,arm64 LDFLAGS:-L${SRCDIR}/build/linux-aarch64
```

Comment:
Was the linux-aarch64 part of this intentional? It's not mentioned in the PR or commit message 😅